docs/modules/user-guide/partials/assembly_migrating-to-devfile-v2.adoc

```diff
@@ -11,7 +11,7 @@ endif::[]
 
 :context: assembly_migrating-to-devfile-v2
 
-This guide explains how to migrate an existing v1.0 devfile to v2.0.
+The following documents explain how to migrate an existing v1.x devfile to a v2.x devfile:
```

Comment:
Is there a reason this was changed from "this guide" to "the following documents"?

Reply:
Yes, I thought "documents" made more sense because Eva pointed out that this document is not a guide; hence, it's not very accurate for us to say 'guide' in this instance.